client/asset/eth/eth.go

```diff
@@ -832,6 +846,18 @@ func (w *ETHWallet) Connect(ctx context.Context) (_ *sync.WaitGroup, err error)
 		}
 	}
 
+	bondAddress, found := dexeth.ETHBondAddress[w.net]
```

Comment:
Polygon will use this connect method too, and we should only be implementing `Bonder` when we have a bond address. 

Please consider this refactor to get the interfaces worked out. https://github.com/buck54321/dcrdex/compare/f374f4c...buck54321:dcrdex:bonder